.github/workflows/process_opened_issues.yml

```diff
@@ -0,0 +1,17 @@
+name: Process (re)opened issues
+
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+
+jobs:
+  add-to-project:
+    name: Adding issue to the master project
```

Comment:
We've generally phased out "master". Can we call this the "main" project instead?

Reply:
Hmm, what do you think of "mega project"? "main project" doesn't inspire. I'm open to other suggestions as well. :)

Reply:
What are we trying to inspire exactly?

Reply:
Well, first off, we want it to be descriptive. "Main" is like "core" in that it's defined relative to other things. It would seem to imply that we have a main project, and other projects that are secondary. While we may end up having some other (likely private) projects for personal use, it's not key to the design. Also, MEGA sounds big and exciting. We deserve to be inspired! :)

Reply:
Hmm I feel like all those criticisms are true of "master" too. Can we just call it the "IREE GitHub project"? If there's only one, then why does it need a name? Serious bikeshedding here, but "mega" sounds to me like we're taking on some tremendous task that is going to have massive budget overruns. Another idea "umbrella project"?

Reply:
> Hmm I feel like all those criticisms are true of "master" too.
This is true. I think "master" was the default name that came up, but hadn't really been through the official branding exercise.

I feel like we're creating a Rorschach test for project management...

Ok, let's call it "IREE" for now, in which case we'll refer to it as the "IREE project" or "IREE Github project". If we need more specificity, we can fix it later. WDYT?